topics/proteomics/tutorials/peptide-library-data-analysis/tutorial.md

```diff
@@ -0,0 +1,185 @@
+>    - *"Datasets"*: `AMPvsTM` 
+>
+{: .hands_on}
+
+
+## Converting tabular data into fasta format
+
+In this step, we will be converting and splitting the tabular data file into fasta format. This tool splits data into two files based on their class labels.
+
+> ### {% icon hands_on %} Hands-on: Converting tabular data into fasta formate
+> 
+> 1. {% tool [PDAUG TSVtoFASTA](toolshed.g2.bx.psu.edu/repos/jay/pdaug_tsvtofasta/pdaug_tsvtofasta/0.1.0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file"*: `PDAUG Peptide Data Access - AMPvsTM (tabular)` (output of **PDAUG Peptide Data Access** {% icon tool %})
+>    - *"Data conversion"*: ` WithClassLabel `
+>
+>
+{: .hands_on}
+
+## Analyzing peptide libraries (AMPs and TMPs) based on features and feature space 
+
+### Summary Plot for peptide libraries 
+
+In this step, we utilize **PDAUG Peptide Sequence Analysis** tool to compare peptide sequences based on hydrophobicity, hydrophobic movement, charge, amino acid fraction, and sequence length and creates a summary plot.
+
+
+
+> ### {% icon hands_on %} Hands-on: Generating a summary plot to assess peptide dataset
+> 
+> 1. {% tool [PDAUG Peptide Sequence Analysis](toolshed.g2.bx.psu.edu/repos/jay/pdaug_peptide_sequence_analysis/pdaug_peptide_sequence_analysis/0.1.0) %} with the following parameters:
+>    - *"Analysis options"*: `Plot Summary`
+>        - {% icon param-file %} *"First input file"*: `PDAUG TSVtoFASTA on data 1 - first (fasta)` (first output of **PDAUG TSVtoFASTA** {% icon tool %})
+>        - {% icon param-file %} *"Second input file"*: `PDAUG TSVtoFASTA on data 1 - second (fasta)` (second output of **PDAUG TSVtoFASTA** {% icon tool %})
+>        - *"first input file"*: `AMP`
+>        - *"Second input file"*: `TM `
+>
+>
+{: .hands_on}
+
+![Summary Plot](../../images/SummaryPlot.png "Summary plot shows comparisoin between AMPs and TMPs")
+
+
+
+### Assessing feature space distribution 
+
+In this tool, we have used **PDAUG Fisher's Plot** that compare two peptide library based on the feature space using the Fisher test. 
+
+
+> ### {% icon hands_on %} Hands-on: Generating a Fisher's plot to assess peptide dataset
+> 
+> 1. {% tool [PDAUG Fisher's Plot](toolshed.g2.bx.psu.edu/repos/jay/pdaug_fishers_plot/pdaug_fishers_plot/0.1.0) %} with the following parameters:
+>    - {% icon param-file %} *"First fasta file"*: `PDAUG TSVtoFASTA on data 1 - first (fasta)` (first output of **PDAUG TSVtoFASTA** {% icon tool %})
+>    - {% icon param-file %} *"Second fasta file"*: `PDAUG TSVtoFASTA on data 1 - second (fasta)` (second output of **PDAUG TSVtoFASTA** {% icon tool %})
+>    - *"Label for first population"*: `AMP`
+>    - *"Label for second population"*: `TM`
+>
+>
+{: .hands_on}
+
+![Fishers plot](../../images/FeatureSpace.png "TMPs peptides show amino acids with larger hydrophobic residues in compare to AMPs")
+
```

Comment:
could benefit from better explanation about what the plots show and what this means, maybe in Form of some Q&A boxes.

Reply:
Q & A added